src/NCronJob/Scheduler/QueueWorker.cs

```diff
@@ -122,10 +122,20 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
     private void AssertUseNCronJobWasCalled()
     {
-        if (!missingMethodCalledHandler.UseWasCalled)
+        if (missingMethodCalledHandler.UseWasCalled)
         {
-            LogUseNCronJobNotCalled();
+            return;
+        }
+
+        if (jobRegistry.GetAllOneTimeJobs().Count == 0)
+        {
+            return;
         }
+
+        throw new InvalidOperationException(
```

Comment:
Maybe putting this bit into the documentation would be nice. And/Or into the xmldocs.

Reply:
Done.